public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -156,6 +181,19 @@ export class PanelEditorUnconnected extends PureComponent<Props> {
     updatePanelEditorUIState({ isPanelOptionsVisible: !uiState.isPanelOptionsVisible });
   };
 
+  onUnlinkPanel = () => {
+    appEvents.emit(CoreEvents.showModalReact, {
+      component: UnlinkModal,
+      props: {
+        onConfirm: () => {
+          delete this.props.panel.libraryPanel;
+          this.props.panel.render();
+          this.forceUpdate();
+        },
+      },
+    });
```

Comment:
would be better to use react or redux & ModalsController  to show modals, using the event is mostly for legacy angular use cases